power_grid_model_c/power_grid_model_cpp/include/power_grid_model_cpp/dataset.hpp

```diff
@@ -43,6 +53,16 @@ class DatasetInfo {
         return handle_.call_with(PGM_dataset_info_total_elements, info_, component_idx);
     }
 
+    Idx component_idx(std::string_view component) const {
```

Comment:
Is this function only for testing purposes? It is only used in the tests

Reply:
well, the entire C++ wrapper is currently only used in tests. So if the argument is to not add it, then we might as well delete the entire C++ wrapper 😉 i do think that this functionality is useful on a more global scale than just in the current tests where it's used, so that's why i added it